cfgov/unprocessed/css/enhancements/forms.less

```diff
@@ -337,6 +343,7 @@ input {
             box-sizing: border-box;
             padding-right: unit( 15px / @base-font-size-px, em );
             width: percentage( 1 / ( @scale-point-count + 1 ) );
```

Comment:
We should change how this is calculated. Losing the extra space that the previous flush treatment afforded is causing this to break onto 3 lines now. I think I'd probably just do something simple like 20% for this and 80% for `.o-scale_answer-options` below.

Reply:
Can you explain what you mean by "break onto 3 lines now", I think I'm missing something.

Reply:
Oh wait, I see what you mean.